### ISSUE TITLE
Host admin can tag Collectives

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#6418]
## User story 
As a Host admin, I want to be able to tag my Hosted Collectives, so I can enable a better picture of the kinds of Collectives in my community. This would also aid the platform to have better data about Collectives across the board.

## Best solution for this problem
A tags box on this screen from the Host admin dashboard > Hosted Collectives section.

<img width="443" alt="Screen Shot 2023-02-14 at 3 33 29 PM" src="https://user-images.githubusercontent.com/5498878/218624300-2491679c-d985-497e-95f1-a05d1efc5d07.png">

Basically, often times Collectives are not motivated to tag themselves and Hosts are willing to do the gruntwork, but right now they lack access.

  P3 high frequency, low impact

**Comment by iamronen** [comment]
Questions for context:
1. What do you mean by "get a better picture" of your collectives?
2. What is it you want to know?
3. What is prompting the question? When does it come up? What are you doing (or trying to do) that is triggering this inquiry?
4. What would that "better picture look like"?
5. What would having that "better picture" make possible for you (that you can't do now)?
6. How are you currently answering this question?

Questions related to tagging:
1. Are you asking for tags separate from the tags collectives can currently use for self identifying? If you are thinking about using the same tags, how do you imagine that working?
2. Which would serve you better: numerous selectable tags or one selected category? This has implications when counting and accounting. Selecting numerous tags means that collectives can be counted multiple times, once in each tag. Selecting one category, please each collective in one clear container.
3. In order to get better information at a platform level "across the board" these (I think) categories need to be shared across all fiscal hosts. How do you imagine going about that?

**Comment by alanna** [comment]
1. In order to enable the discovery dashboard to see what Collective categories OCF has, we needed to add a bunch of tags. We did this manually with the help of the dev team, but all host admins should have access to do this. 
2. E.g. Gift Collective wants to know how many Maori Collectives vs Climate Collectives etc it has, and we do know, but without tagging them it's hard to get a full picture and the software can't support us to know our community.
3. Marketing, fundraising, strategic planning, onboarding (Collectives want to know who else is here)
4. Basically the discovery dashboard is great, but it doesn't work if Collectives aren't tagged comprehensively
5. Reporting, marketing, planning, programs, service development, onboarding
6. We're not (besides OCF, who did a one off manual tagging exercise that we have no tools to keep maintained now)

1. No I think the same tags
2. numerous tags, I think (e.g. Christchurch and Climate)
3. True, I don't know

**Comment by kaylarep** [comment]
I have major privacy concerns about this. I really do not think we should be tagging collectives. There is so much nuance involved, and if we mess this up (using a label for a collective they would not use for themselves), there are major implications that could arise down the line.

**Comment by kaylarep** [comment]
Also, addressing this issue is 'high impact'. 

Without this tagging prioritized, the 'Discovery Dashboard' (a tool that we are getting good feedback on) would be pretty useless...

**Comment by alanna** [comment]
I can see the issue with forcing tags on Collectives being bad in some cases. I guess it's just on hosts to manually chase them to add tags if they want to be able to get a picture of their community as a whole, or if the platform wants that picture, but that's just never going to work to some degree. Maybe if we make the benefits of having tags really strong and clear, Collectives would be more motivated?

I do think we need to allow multiple tags though. How would we decide whether to tag a Collective `kaupapa maori` or `climate` if it's both? These are both key categories for Gift Collective, for example. Same with `food` and `mutual aid` and any number of others for OCF.

**Comment by Betree** [comment]
This has been requested again over the past few years.

Recently there is one use case this feature who would have proven to be useful: Open Collective Europe want to categorize their collectives in three tiers in order to assign them to different members of their team. Having a tag system at the collective level that stays private and usable just by the host admins would have allowed them to rely on the platform to achieve that.
